modules/sap_hana/variables.tf

```
@@ -15,17 +15,22 @@
  */
 
 variable "project_id" {
-  description = "The ID of the project in which the resources will be deployed."
+  description = "The ID of the service project in which the resources will be deployed."
```

Variables should have types.

```suggestion
  description = "The ID of the service project in which the resources will be deployed."
  type        = string
```